topics.go

```diff
@@ -7,3 +7,6 @@ package mainflux
 
 // OutputSenML represents subject SenML messages will be published to.
 const OutputSenML = "out.senml"
+
+// InputChannels represents subject all messages will be published to.
```

Comment:
I wouldn't be mean on words here, its very important to explain clear and simple. I would explain what is input channel, how input message looks like (reference), what is output channel and what is payload on output, etc..